### PR TITLE
Pixel: revise single template

### LINKED ISSUE #6461: Pixel: Styles: Quote
Update quote pattern.

Design: Cm51iCQSPu8Ar1qufDTeOQ-fi

image

### LINKED ISSUE #6465: Pixel: Templates: Single
Update single page template.
Comments block styles will be done here #6459 

Designs: Cm51iCQSPu8Ar1qufDTeOQ-fi

image

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

Revises Pixel's single template, and implements the quote pattern. 

screencapture-gutenbergtest-local-what-was-that-song-again-2022-09-06-13_48_07

#### Related issue(s):

#6465 #6461

### CODE REVIEW

File: pixel/templates/single.html [hunk 1]
```diff
@@ -1,28 +1,16 @@
...
-<!-- /wp:group -->
-
-<!-- wp:group {"tagName":"main","lock":{"move":false,"remove":true}} -->
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained"},"lock":{"move":false,"remove":true}} -->
 <main class="wp-block-group">
-<!-- wp:post-featured-image {"align":"full"} /-->
-
+<!-- wp:post-featured-image {"style":{"border":{"width":"2px"}},"borderColor":"primary","className":"pixel-shadow"} /-->
+<!-- wp:post-title /-->
 <!-- wp:post-content {"layout":{"inherit":true},"lock":{"move":false,"remove":true}} /-->
```
Comment (madhusudhand) [inline pixel/templates/single.html]: small issue with left alignment in small devices.

image

File: pixel/patterns/quote.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ * Title: A Pixel Quote
+ * Slug: pixel/quote
+ * Categories: Featured, Text
+ */
+
+?>
+
+<!-- wp:pullquote {"style":{"border":{"width":"2px"},"typography":{"fontStyle":"normal","fontWeight":"400","lineHeight":"1.4","letterSpacing":"-0.04em","fontSize":"1.25rem"}},"borderColor":"primary","className":"pixel-shadow"} -->
+<figure class="wp-block-pullquote pixel-shadow has-border-color has-primary-border-color" style="border-width:2px;font-size:1.25rem;font-style:normal;font-weight:400;letter-spacing:-0.04em;line-height:1.4"><blockquote><p><?php echo esc_html__( '"The pain itself is love, the main storage system. Aenean dapibus nisi quis dolor scelerisque egestas. Members of the mass need to feel comfortable and easy. It is important that the lion should invest."', 'pixel' ) ?></p><cite></cite></blockquote></figure>
+<!-- /wp:pullquote -->
```
Comment (madhusudhand) [inline pixel/patterns/quote.php:12]: I guess because of the empty `cite` element pattern needs recovery. Add default text as `Unknown`

image

Comment (madhusudhand) [reply pixel/patterns/quote.php:12]: And update the letter spacing to `-0.02em`.

File: pixel/templates/single.html (hunk 1 above)
Comment (madhusudhand) [reply pixel/templates/single.html]: Images (default, wide and full width) all set to same size in frontend. But they work good in editor.

| Frontend | Editor |
| --- | --- |
| image | image |

File: pixel/patterns/quote.php (hunk 2 above)
Comment (jffng) [reply pixel/patterns/quote.php:12]: Done in [114a0e8](https://github.com/Automattic/themes/pull/6513/commits/114a0e8f4ca326eea2da48a1a46c4e6a80b25409).

File: pixel/templates/single.html (hunk 1 above)
Comment (jffng) [reply pixel/templates/single.html]: I think I've addressed this in [a954f79](https://github.com/Automattic/themes/pull/6513/commits/a954f7901793284a312352eae89a6f41dde1a9c2), mind giving this another test?

**Review by madhusudhand — APPROVED** [review]
Looks good.

**Pull request merged**